internal/bowtie/resources/dns.go

```diff
@@ -318,6 +347,8 @@ func (d *dnsResource) Update(ctx context.Context, req resource.UpdateRequest, re
 		return
 	}
 
+	// servers := mergeServerDetails(plan.Servers, )
```

Comment:
Just checking, you want to keep this around for later?

Reply:
Nope I will remove it